### PR TITLE
feat(sbb-flip-card): add flip event and isFlipped getter

### LINKED ISSUE #2961: [Feature request]: Add a Event to dispatch the toogle state
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Request type

Request for a new component

### It affects the following packages

Components

### Functionality

It must be possible to record or read out the status of the Toogle. The application scenario consists of the tracking that is to be carried out when the user clicks open and close (toogle).

@sbb-esta/lyne-elements/flip-card
fix: add a Event to dispatch the status

### Link to design proposal file

_No response_

Edit by @kyubisation:

Tech Spec:

- Add getter isFlipped: boolean
- Add event `flip`, which is dispatched every time the flip card is flipped (each direction)

### PR DESCRIPTION
**Pull request by dauriamarco**
Closes #2961

### CODE REVIEW

File: src/elements/flip-card/flip-card/flip-card.ts [hunk 1]
```diff
@@ -33,6 +41,11 @@ export class SbbFlipCardElement extends SbbHydrationMixin(LitElement) {
     return this.querySelector('sbb-flip-card-details')!;
   }
 
+  /** Returns whether the flip card is flipped. */
```
Comment (jeripeierSBB) [inline src/elements/flip-card/flip-card/flip-card.ts:44]: nitpick
```suggestion
  /** Whether the flip card is flipped. */
```

**Review by jeripeierSBB — APPROVED** [review]
LGTM

File: src/elements/flip-card/flip-card/flip-card.spec.ts [hunk 2]
```diff
@@ -56,6 +61,9 @@ describe('sbb-flip-card', () => {
     element.toggle();
     await waitForLitRender(element);
 
+    await waitForCondition(() => flipSpy.events.length === 1);
```
Comment (jeripeierSBB) [inline src/elements/flip-card/flip-card/flip-card.spec.ts:64]: You could add a check for isFlipped to get code coverage happy ;)

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Review by MarioCastigliano — APPROVED** [review]
All good 👍

**Pull request merged**